modules/javafx.controls/src/main/java/javafx/scene/control/Skin.java

```diff
@@ -60,6 +71,23 @@
      */
     public Node getNode();
 
+    /**
+     * Called by {@link Skinnable#setSkin(Skin)} after the
+     * previous skin, if any, has been uninstalled via its {@link #dispose()} method.
+     * This method allows a Skin to register listeners, add child nodes, set
+     * required properties and/or event handlers.
+     * <p>
+     * The default implementation of this method does nothing.
+     *
+     * @implNote
+     * Most implementations of Skin in the <code>javafx.controls</code> module
```

Comment:
If this is only targeting controls in the javafx.controls module, it is not relevant to developers using JavaFX.

Reply:
It might be best to remove `in the <code>javafx.controls</code> module`, since this advice is relevant for third-party controls as well.